### DISCUSSION TITLE
What would public photo/album mean in TripUp App?

### DISCUSSION THREAD

**Discussion by lxs** [tripupapp/tripup-server#7] [General]
**Discussion Topic:** Does "Non identified recipient" == Public Album/Photo? perhaps not always.

If the creator wants to share an endpoint without specifying an identifiable recipient then in many cases this might be deemed the equivalent to setting as public access (albeit perhaps with limited obscurity). Are there examples where this is not a valid scenario?

If "non-identified recipient" is the equivalent to "public domain": Why not enable this as a feature, allowing an Album to be set public (=anyone with URL). 
-- In this case the URL with key would be in public domain (yet should still grant the creator the option to revoke access). 
-- However, there could be layered approach to this: album could be accessible to anyone who has URL to the endpoint, yet not visible on public profile of the creator. 

**Furthermore:** the creator could set the album to be visible on their profile, yet set access to public or enable the option for access request (by an identifiable recipient).

**Comment by vin047** [comment]
Hmm, the way I see it, if you want to make an entire album public, you'd be better of using Instagram (or a userdata respectful alternative to Instagram). I'm not sure I see the point in encrypting photos/albums only to then make them public. Or have I misunderstood something? Is there a use case that i'm not seeing?

**Reply by lxs** [reply]
@vin047 apologies for the extensive delay in replying. Had not been attentive to notification of your response...

The gist was about making it easy to share photo album without the burden of specifying the recipients' user IDs (1 or more).

Recall the time of earlier **Google Photo Albums** which could be "secured" to a group defined via Google Contact labels. 
- The authentication + group access being handled by Google... + notifications. 
- As such, via Google Contacts, I might have "family" label, "university friends" label, "close friends" label, etc -- and be able to share Google Albums without those albums being accessible to those who were not explicitly members of a defined group that had been granted access to the album. 
- Those were comparatively simpler times.......... [also, photos tended to be smaller and internet speeds slower....]

-//-

Perhaps I can seek to explain via a scenario:
- Parents of a new-born wish to share photos of their infant with their family + closest friends, yet do not wish to place these photos in public domain for others (e.g. search engines / AI / random folks). 
- Let's say they wish to preserve privacy and the image copyright, yet do not wish to have headache of defining permissions - nor send via WhatsApp/Signal to a group or limited distribution list.
- Use the scenario of new infant photos as such images can be hugely private, yet also superbly gleeful to share with a trusted community...
- ...and if those trusted community members opted to download a photo - then it would have copyright and/or open access license defined in the meta data of the file, and optionally watermarked on the downloaded image... (even more awesome would be to have option to apply secure Steganography = to embed copyright ownership + license)

-//-

Today, I might explore use of capabilities from Mastodon or Pixelfed (and yourself, naturally!) -- with a view to establishing trust community/communities that could share photographs within the (decentralised) trust network...
